requirements.txt

```diff
@@ -1,6 +1,8 @@
+cytominer-database>=0.3.4
```

Comment:
We specifically made this an optional requirement. (see #160)

Do we need to add it here?

Reply:
I missed the notes in the `readme.md`, my apologies. I had added the dependency due to having a base installation of pycytominer without the collate extra and running into errors with test_collate as a result. I'll back this out so as to not cause any complications here.

Reply:
Just a note here: dependency removed in a further commit.